<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>grade_calculator</class>
 <widget class="QMainWindow" name="grade_calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>448</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>grade_calculator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>250</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QSlider" name="hw6_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw6_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_9">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>330</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QSlider" name="hw8_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw8_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="SchemaA">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema A</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_7">
    <property name="geometry">
     <rect>
      <x>276</x>
      <y>160</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QSlider" name="m2_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>50</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QSlider" name="hw1_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw1_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>130</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QSlider" name="hw3_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw3_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>290</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QSlider" name="hw7_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw7_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>70</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QSlider" name="m1_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>210</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QSlider" name="hw5_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw5_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="SchemaB">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Schema B</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="m2_sbox">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QToolButton" name="OverallButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Overall Score:</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>90</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="hw2_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw2_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_10">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>170</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QSlider" name="hw4_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="hw4_sbox">
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_11">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>250</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QSlider" name="final_slider">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QSpinBox" name="final_sbox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>280</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="m1_sbox">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QComboBox" name="ClassScroller">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>10</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PIC 10C</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Math 115A</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>HW8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="namebutton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Course Grade Calculator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="coursebutton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Course:</string>
    </property>
   </widget>
   <widget class="QToolButton" name="m1button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 1</string>
    </property>
   </widget>
   <widget class="QToolButton" name="m2button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Midterm 2</string>
    </property>
   </widget>
   <widget class="QToolButton" name="finalexambutton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Final Exam</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_score">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>489</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>hw1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>99</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw1_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw1_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>196</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw2_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw2_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw3_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw3_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>151</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>204</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw4_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw4_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>156</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw5_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw5_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>200</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>294</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw6_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw6_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>294</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>333</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw7_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw7_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>341</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw8_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw8_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>369</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hw8_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>hw8_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m1_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m1_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m1_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m1_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m2_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m2_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>366</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m2_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>m2_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>392</x>
     <y>238</y>
    </hint>
    <hint type="destinationlabel">
     <x>392</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>final_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>final_sbox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>final_sbox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>final_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>340</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>337</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
